### PR TITLE
[WIP] added __version__ attribute for python library 

### LINKED ISSUE #1339: [Feature Request]
### Overview

Add delta lake library version to python api.

I am unable find installed delta library version programmatically, when using python package.

let me know if feature already exists, i didn't find it.

### Willingness to contribute

The Delta Lake Community encourages new feature contributions. Would you or another member of your organization be willing to contribute an implementation of this feature?

- [ ] Yes. I can contribute this feature independently.
- [x] Yes. I would be willing to contribute this feature with guidance from the Delta Lake community.
- [ ] No. I cannot contribute this feature at this time.

### PR DESCRIPTION
**Pull request by gokulyc**
## Description
Resolves #1339 

It version attribute to python library.  

## Does this PR introduce _any_ user-facing changes?

Add \_\_version\_\_ attribute with python library version value extracted from version.sbt

### CODE REVIEW

File: python/delta/_version.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from pathlib import Path
+# delta.io version
+def get_version_from_sbt():
+    version = Path("../../version.sbt").read_text().strip()
```
Comment (gokulyc) [inline python/delta/_version.py:4]: @nkarpov after I found a problem here. It won't work.

After creating python package version.sbt won't be available at specified. version.sbt or similar should be added python source path.

**Comment by zsxwing** [comment]
@gokulyc do you just want to know the Delta Lake version when running in Databricks? Otherwise, you can just run the following code to get it.

```
import importlib_metadata
importlib_metadata.version("delta_spark")
```

**Comment by gokulyc** [comment]
i am databricks runtime 11.0


above snippet is throwing error.

- importlib_metadata package is not there

python version of runtime is 3.9.5

when i also tried below snippet

from importlib.metadata import version
version ("delta_spark")

**Comment by gokulyc** [comment]
Thanks @zsxwing I found https://pypi.org/project/importlib-metadata/ . this is working in my local machine.

working code: 

`from importlib_metadata import version

VERSION = version("delta-spark")`

Still in databricks its not working though (even in output of %pip list , delta-spark is not available)

I think in databricks docs(https://docs.databricks.com/release-notes/runtime/11.1.html) is only way know installed version.

**Comment by zsxwing** [comment]
Databricks Runtime is using a different way to pack delta lake and ship it. A version X.Y of Delta Lake in Databricks Runtime release note means this Databricks Runtime version supports all of public APIs existing in Delta Lake X.Y, but technically that's a super set rather than the exact version because of different release cadences.

**Pull request closed without merging**